AnalyticRules/Data_Alert.yaml

```diff
@@ -0,0 +1,28 @@
+id: "1d2c3da7-60ec-40be-9c14-bade6eaf3c49"
+name: "Data Alert"
+description: |
+  'Data Compromised'
+severity: Medium
+status: Available
+requiredDataConnectors: []
+queryFrequency: "5m"
+queryPeriod: "5m"
+triggerOperator: "gt"
```

Comment:
This applies to all the analytical rules added in this PR

Please remove double quotes from` queryFrequency, queryPeriod, triggerOperator`

For reference, please check this analytical rule :- https://github.com/Azure/Azure-Sentinel/blob/df13d3eff814823b953a190e94e95aea3e481bf9/Solutions/Azure%20Active%20Directory/Analytic%20Rules/FailedLogonToAzurePortal.yaml#L4